saleor/core/utils/promo_code.py

```diff
@@ -9,14 +9,19 @@
 class InvalidPromoCode(ValidationError):
     def __init__(self, message=None, **kwargs):
         if message is None:
-            message = {"promo_code": "Promo code is invalid"}
+            message = {
+                "promo_code": ValidationError("Promo code is invalid", code="invalid")
+            }
         super().__init__(message, **kwargs)
 
 
 class PromoCodeAlreadyExists(ValidationError):
     def __init__(self, message=None, **kwargs):
+        code = kwargs.get("code", "promo_code_already_exists")
```

Comment:
Shouldn't we use `ErrorCode` class here?

Reply:
As above.